billboardd/daemon.h

```diff
@@ -295,6 +303,13 @@ class Daemon : public QObject {
             context->subscribeTo(interestingProperties);
             gconfmon->subscribeTo(interestingProperties);
 
+            if (interestingProperties.contains("tklock")) {
+                QObject::connect(&tklock, SIGNAL(locked()),
```

Comment:
ok, cool. that was my original intention; i thought you wanted to make this opt-in using the properties. ill remove them now.

also, how do you feel about NOT re-rendering while the screen is unlocked? {i dont know if you already handle this elsewhere}